src/umpire/ResourceManager.cpp

```diff
@@ -130,7 +130,7 @@ ResourceManager::initialize()
 #endif
 
 #if defined(UMPIRE_ENABLE_CUDA)
-  m_memory_resources[resource::Constant] = registry.makeMemoryResource("DEVICE_CONST", getNextId());
+  //m_memory_resources[resource::Constant] = registry.makeMemoryResource("DEVICE_CONST", getNextId());
```

Comment:
Please maintain this change locally. As you know, we haven't been able to reproduce the issue.